backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -26,10 +26,29 @@ include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+1]
 include::modules/oadp-using-data-mover-for-csi-snapshots.adoc[leveloffset=+1]
 
+[id="oadp-12-data-mover-ceph"]
+== Using OADP 1.2 Data Mover with Ceph storage
+
+You can use OADP 1.2 Data Mover to backup and restore application data for a cluster that uses CephFS, a cluster that uses CephRBD, and for a cluster that uses both.
+
+OADP 1.2 Data Mover leverages some of the recently added features of Ceph that support large scale environments, one of which is the shallow copy method, which is available for {product-name} 4.12 and later. This feature supports backing up and restoring `StorageClass` and `AccessMode` resources other than what is found on the source PVC.
+
+This document has the following sections:
+
+* Prerequisites
+* Preparing Ceph `VolumeSnapshotClass` and `StorageClass` CRs for use with OADP 1.2 Data Mover
```

Comment:
Looks like there is a small build error according to Jenkins - 
```
Validating backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc. Validation will fail with FAILED, ERROR, or WARNING messages...
```
```
asciidoctor: WARNING: modules/oadp-deleting-backups.adoc: line 39: unterminated listing block
```

Reply:
@anarnold97 I have gone over the file oadp-deleting-backups.adoc and cannot find any problem with its blocks or the rest of it. I even copy-pasted the file from "main" and the error persisted. The file now appears in the set of files affected because of that copy-paste, but, as I said, the file is still not OK. Could you please take a look? The assembly that calls it looks OK.

Reply:
Thanks very much leave it with me. I will get to it later